gulpfile.js

```diff
@@ -825,17 +826,32 @@ function checkBinarySize(compiled) {
   const file = compiled ? './dist/v0.js' : './dist/amp.js';
   const size = compiled ? '77.05KB' : '336.66KB';
   const cmd = `npx bundlesize -f "${file}" -s "${size}"`;
-  log(green('Running ') + cyan(cmd) + green('...\n'));
-  const p = exec(cmd);
-  if (p.status != 0) {
-    log(red('ERROR:'), cyan('bundlesize'), 'found that amp.js/v0.js has ' +
-        'exceeded its size cap. This is part of a new effort to reduce ' +
-        'AMP\'s binary size (#14392). Please contact @choumx or @jridgewell ' +
-        'for assistance.');
-    // Terminate Travis builds on failure.
-    if (process.env.TRAVIS) {
-      process.exit(p.status);
+  if (!process.env.TRAVIS) {
+    log('Running ' + cyan(cmd) + '...');
+  }
+  const output = getStdout(cmd);
+  const pass = output.match(/PASS .*/);
+  const fail = output.match(/FAIL .*/);
+  const error = output.match(/ERROR .*/);
+  if (error) {
+    log(yellow(error));
+  } else if (fail) {
```

Comment:
I'm curious how much value we're getting from `bundlesize` when we're parsing its output.

Something to consider in the future... should we create something more constrained to our needs?

Reply:
The only reason I'm parsing its output is to silence the annoying warning / suggestion mentioned in the PR description (item 2). We can't implement native PR integration because AMP's PRs come in from forks, and Travis doesn't allow secure tokens to be used.

`bundlesize` is a widely used tool, so I still think there's benefit in using it.